controller/src/main/java/io/pravega/controller/store/stream/PravegaTablesStream.java

```diff
@@ -812,6 +812,25 @@ private String getWritersTableName(String id) {
         });
     }
 
+    @Override
+    public CompletableFuture<Map<UUID, TxnStatus>> listCompletedTxns(final OperationContext context) {
+        Preconditions.checkNotNull(context, "operation context cannot be null");
+        return getTxnInBatch(0, context);
+    }
+
+    private CompletableFuture<Map<UUID, TxnStatus>> getTxnInBatch(final Integer batch, final OperationContext context) {
+        Preconditions.checkNotNull(context, "operation context cannot be null");
+
+        Map<String, VersionedMetadata<CompletedTxnRecord>> result = new ConcurrentHashMap<>();
+        String tableName = getCompletedTransactionsBatchTableName(batch);
+        return storeHelper.expectingDataNotFound(storeHelper.getAllEntries(
+                tableName, CompletedTxnRecord::fromBytes, context.getRequestId()).collectRemaining(x -> {
```

Comment:
is it possible to provide a better name to these variables isntead of `v`, `x`?

Reply:
Renamed x. V is referring to void so keep it same